### ISSUE TITLE
Possible issue with PyYAML's load()

### ISSUE THREAD

**Issue by nkantar** [python/typeshed#2886]
I'm running into a confusing error when using mypy to check a project using PyYAML.

Relevant code:

```python
raw = yaml.load(some_str, Loader=yaml.FullLoader)
```

Error message (from `mypy --strict`):

```
error: Call to untyped function "load" in typed context
```

Yet the [typeshed](https://github.com/python/typeshed/blob/master/third_party/2and3/yaml/__init__.pyi#L18) has:

```python
def load(stream: Union[str, IO[str]], Loader=...) -> Any: ...
```

Versions:
• OS: macOS 10.14.3
• Python: 3.7.2 (installed via Homebrew)
• Pipenv: 2018.11.26
• mypy: 0.670
• PyYAML: 5.1

(If this belongs in the [mypy repo](https://github.com/python/mypy), I'm happy to move it.)

**Comment by JelleZijlstra** [comment]
This might be because `Loader` is untyped in typeshed.

**Comment by adamtheturtle** [comment]
This was fixed in #3035.

**Comment by AMWJ** [comment]
This doesn't appear resolved? #3035 adds no types to the `load()` method's `Loader` argument, so `Loader`'s type remains Unknown.

It appears #1657 attempted to resolve this incorrectly, and #1695 reverted. I think this should be re-opened until `load()` and `load_all()` are typed.

**Comment by Akuli** [comment]
Feel free to submit a PR :)

**Comment by adamtheturtle** [comment]
Apologies for my mistake.

**Comment by Avasam** [comment]
Is this fixed by #9752 ?

**Comment by AlexWaygood** [comment]
> Is this fixed by #9752 ?

Looks like it!
